intuita-webview/src/main/SearchBar/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { VSCodeTextField } from '@vscode/webview-ui-toolkit/react';
+import cn from 'classnames';
+import { Dispatch, SetStateAction } from 'react';
+import styles from './style.module.css';
+
+type Props = {
+	searchQuery: string;
+	setSearchQuery: Dispatch<SetStateAction<string>>;
+};
+
+export const SEARCH_QUERY_MIN_LENGTH = 3;
+
+const SearchBar = ({ searchQuery, setSearchQuery }: Props) => {
+	return (
+		<VSCodeTextField
+			type="text"
+			value={searchQuery}
+			placeholder=""
+			onInput={(event: any) => {
```

Comment:
is this `event` typed by the VSCodeTextField?

Reply:
Nope, I already checked ;(
![Screenshot 2023-04-25 at 21 49 17](https://user-images.githubusercontent.com/32841130/234400232-457e2d88-ac62-4ca6-8dd5-7e4e84b15164.png)